meerkat_abacus/consumer/get_data.py

```diff
@@ -15,14 +18,32 @@ def read_stationary_data(get_function, param_config, celery_app, N_send_to_task=
                          "data": dict(element)})
             if i % N_send_to_task == 0:
                 logging.info(f"Processed {i} records")
-                celery_app.send_task("processing_tasks.process_data", [data])
+                send_task(data, celery_app, celery_inspect)
                 data = []
         if data:
+            send_task(data, celery_app, celery_inspect)
             logging.info("Cleaning up data buffer.")
-            celery_app.send_task("processing_tasks.process_data", [data])
         logging.info("Finished processing data.")
         logging.info(f"Processed {i} records")
 
+
+def send_task(data, celery_app, inspect, N=20):
+    """
+    Sends data to process queue if the the are less than N tasks waiting
+
+
+    """
+    inspect_result = inspect.reserved()["celery@abacus"]
+    logging.info(inspect_result)
```

Comment:
I'd be a more descriptive here.
`logging.info(f"Celery inspect result: {inspect_result}")` or sth simillar.